content/operator/cd/index.de.md

```diff
@@ -144,19 +144,34 @@ Für reguläre Fahrkarten reisen Kinder bis 6 Jahre in der 2. Klasse kostenlos.
 
-Übersicht über die Verbindungen: [ČD kommerzielle Zuge und ausgewahlte Linien 2024-2025](https://www.raildeliverygroup.com/images/RST/CD%20kommerzielle%20Zuge%20und%20ausgewahlte%20Linien%202024-2025.pdf)
+Der Aufschlag kostet ~125 CZK (Stand November 2025) und kann vor Fahrtantritt am ČD Ticketschalter oder im Zug (bitte das Zugpersonal direkt aufsuchen) gekauft werden. Die Gültigkeit des Aufschlags entspricht der gleichen Dauer wie das aktuelle Feld auf dem FIP Freifahrtschein (2 Tage) und ist in allen kommerziellen Verbindungen gültig. [^3]
 
 Rückerstattungen von Aufschlägen sind bis 23:59 Uhr am Tag vor dem ersten Gültigkeitstag möglich (ohne Abzug) oder innerhalb von 15 Minuten nach dem Kauf (ohne Abzug). Wird ein unbenutzter Aufschlag bis 08:00 Uhr am ersten Gültigkeitstag zurückgegeben, wird eine Bearbeitungsgebühr von 100 CZK erhoben. In allen anderen Fällen ist keine Rückerstattung möglich. [^1]
 
+#### Züge mit Reservierungspflicht
+
+Einige Züge der ČD haben eine Reservierungspflicht. Dies umfasst alle `SC` sowie einige `IC` Züge (alle Verbindungen sind aufgeführt in den [kommerziellen Verbindungen](#čd-kommerzielle-verbindungen-aufschlagspflichtig)). Die betroffenen `IC` Züge verkehren primär auf der Strecke Prag – Ostrava.
+
+Diese Züge benötigen eine spezielle Sitzplatzreservierung (inkl. Aufschlag). Die ČD hat dafür zur Zeit keinen eigene Tarifoption. Folgende Möglichkeiten für die Sitzplatzreservierung werden in der Regel akzeptiert: [^3]
+
+- Der Kauf einer Reservierung für ČD Personal (Tarif: _Rail card ČD_) über die Website der ČD ([ČD eshop](https://www.cd.cz/en/eshop/default.htm)) oder die ČD App (Můj vlak)
+- Der Kauf einer herkömmlichen Reservierung (siehe [Ticket- und Reservierungskauf](#ticket--und-reservierungskauf)) und zusätzlich des [kommerziellen Aufschlags](#čd-kommerzielle-verbindungen-aufschlagspflichtig)
+
+Vor dem Einstieg in den Zug kann es hilfreich sein das Zugbegleitpersonal zu informieren.
+
 ### Verbindungen der ČD ohne FIP
 
-Grundsätzlich gilt FIP theoretisch in keinen ČD Zügen, die im Rahmen von gewonnenen öffentlichen Ausschreibungen verkehren. Leider stellt die ČD keine vollständige Liste zur Verfügung. Die wichtigsten sind jedoch die folgenden:
+Grundsätzlich gilt FIP theoretisch in keinen ČD Zügen, die im Rahmen von gewonnenen öffentlichen Ausschreibungen verkehren und nicht wenigstens ein Teilabschnitt direkt durch das Verkehrsministerium vergeben wurde. Überregionale Verbindungen, in denen kein FIP gilt, werden auch [kommerzielle Verbindungen](#čd-kommerzielle-verbindungen-aufschlagspflichtig) genannt. Die ČD stellt hierfür eine [Liste](#čd-kommerzielle-verbindungen) zur Verfügung, in der die betroffenen Verbindungen aufgeführt sind. Züge der Kategorie `EC`, `RJ` und `R` (Ausnahme Züge auf der Nato-Bahn) sind in der Regel nicht betroffen und können innerhalb Tschechiens immer mit FIP genutzt werden.
```

Comment:
Wo stehen in der Liste denn die Verbindungen, für die FIP gar nicht akzeptiert wird? Ich sehe nur die aufschlagpflichtigen Verbindungen, aber das ist ja nochmal ein Unterschied zu gar keiner Akzeptanz.